embedded-graphics/src/draw_target.rs

```diff
@@ -100,8 +107,18 @@ use crate::{
 ///
 /// In addition to defining [`draw_pixel`], an implementation of [`DrawTarget`] can also provide
 /// alternative implementations for hardware accelerated drawing operations. This example implements
```

Comment:
This might be misleading, because "alternative implementations" could be interpreted as alternative implementations of `draw_pixel`.

Reply:
Yeah maybe it's a bit vague, but IMO reading on to the next paragraphs and looking at the code example clarifies things. Can we leave this as is for now? Docs are a simple addition to future releases. Non breaking changes from e-g? Who'd've thought???!!1/!?!?1/1/1 :grin:

Reply:
> Can we leave this as is for now?

OK.

> Non breaking changes from e-g? Who'd've thought???!!1/!?!?1/1/1 grin

That's a novel idea, but I promise I'll find more ways to break the code in future releases :wink:

Reply:
> That's a novel idea

I'm gonna patent it

> I promise I'll find more ways to break the code in future releases 

Wahey but that's what semver pre-1.0.0 is for :+1: